Configure

```diff
@@ -2628,7 +2628,7 @@ return(0); }
 	LSOF_TSTBIGF=" "
 	LSOF_VERS="1006000"
 	;;
-      1*)
+      1.*)
```

Comment:
Seeing how for 2.x, 3.x, 4.x, and 5.x this is handled just as `2*`, `3*`, `4*`, and `5*` respectively, I'd suggest to update those places to use `2.*`, `3.*`, `4.*`, and `5.*` instead.